### ISSUE TITLE
Spread operator causing `check validity of block` error

### ISSUE THREAD

**Issue by claudiamatosa** [sasstools/sass-lint#837]
Hey guys,

I ran into an error when trying to lint a mixin that uses the spread operator for a parameter. I've tried enforcing the `scss` syntax when running sass-lint, but to no avail. Is there anything that I can do to avoid it?

Cheers!
### Error

```
24:1  error  Please check validity of the block starting from line #24  Fatal
```

This is thrown on the line where the spread operator is.
### Code

```
@mixin transition(
  $transition-duration: default,
  $transition-timing-function: default,
  $transition-property...
) {
  // Mixin code
}
```
### Stacktrace

```
(...)node_modules/sass-lint/index.js:294
    throw new Error(errorCount.count + ' errors were detected in \n- ' + errorCount.files.join('\n- '));
    ^

Error: 1 errors were detected in
- app/Resources/assets/styles/mixins/_transition.scss
    at Function.sassLint.failOnError ((...)node_modules/sass-lint/index.js:294:11)
    at detectPattern ((...)node_modules/sass-lint/bin/sass-lint.js:33:10)
    at (...)node_modules/sass-lint/bin/sass-lint.js:98:5
    at Array.forEach (native)
    at Object.<anonymous> ((...)node_modules/sass-lint/bin/sass-lint.js:97:16)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
```
### What version of Sass Lint are you using?

1.9.0
### Please include any relevant parts of your configuration

Running with command line:
Tried `sass-lint styles/**/*.scss` and `sass-lint styles/**/*.scss  --syntax scss`

**Comment by DanPurdy** [comment]
Hey @claudiamatosa This is an issue with gonzales-pe the AST that we use to parse Sass/scss. Please could you repost this error there? Thanks.

https://github.com/tonyganch/gonzales-pe/issues

**Comment by claudiamatosa** [comment]
Thanks, @DanPurdy, will do. Just fyi, while trying to recreate the error using only `gonzales-pe`, I pinned the issue to something more specific: it only occurs if the previous arguments have default values. This means the the following will not throw an error:

```
@mixin transition(
  $transition-duration,
  $transition-timing-function,
  $transition-property...
) {
  // Mixin code
}
```

**Comment by DanPurdy** [comment]
Great, that's really helpful for us when we go to look at this. Thanks!

**Comment by bgriffith** [comment]
Thanks @claudiamatosa! Trust you to find a bug haha! 😁

**Comment by dovidweisz** [comment]
The referenced bug is now fixed, but your import of "gonzales-pe" is fixed at  "3.4.7"

**Comment by DanPurdy** [comment]
Hi @dovidweisz we know, we're waiting for a few more issues to be fixed and released before we update sass-lint to use the new version of Gonzales, there's a few breaking changes we need to handle in order to support this so unfortunately there will need to be a little bit of waiting.

**Comment by bgriffith** [comment]
This is fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!
